### PR TITLE
ADM-550 [Fox][frontend] - Real done reset issue 

### PR DESCRIPTION
**Pull request by ZhiqiaoYang**
## Summary

Only when the Cycle time component marks one column as done, the Real down component will be reset.

## Before

Whenever the Cycle time component updates, the Real down component resets.

**Screenshots**
Screenshot 2023-08-08 at 17 52 10

## After

_Description_
Only when the Cycle time component marks one column as done, the Real down component will be reset.

**Screenshots**
Options change to Done, will reset:
Screenshot 2023-08-08 at 17 50 28
Done change to other options, will reset:
Screenshot 2023-08-08 at 17 51 08
Options except Done change to other options, will not reset:
Screenshot 2023-08-08 at 17 51 35

## Note

_Null_

### CODE REVIEW

File: frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx [hunk 1]
```diff
@@ -33,11 +35,23 @@ export const CycleTime = ({ title }: cycleTimeProps) => {
...
+    } else if (saveDone.includes(name)) {
+      setSaveDone(saveDone.filter((e) => e !== name))
+      dispatch(saveDoneColumn([]))
+    }
   }
 
   useEffect(() => {
     dispatch(saveCycleTimeSettings(cycleTimeOptions))
+    cycleTimeOptions.forEach((item) => {
+      if (item.value === 'Done') {
```
Comment (ziwen-xiao) [inline frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx:51]: seems we have defined the constant 'Done' in constants.ts, probably let's use the constant here as well to maintain consistency.

**Review by ziwen-xiao — COMMENTED** [review]
BTW, a small typo issue of this pr title should be fixed (Real done)

**Review by guzhongren — APPROVED** [review]

File: frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx (hunk 1 above)
Comment (ZhiqiaoYang) [reply frontend/src/components/Metrics/MetricsStep/CycleTime/index.tsx:51]: Fixed, thanks ziwen!

**Review by ziwen-xiao — APPROVED** [review]
LGTM

**Pull request merged**